lib/modules/solidity/solcP.js

```diff
@@ -1,5 +1,18 @@
 let solc;
 
+let fs = require('fs-extra');
+let path = require('path');
+
+function findImports(filename) {
+  if (fs.existsSync(filename)) {
+    return {contents: fs.readFileSync(filename).toString()};
+  }
+  if (fs.existsSync(path.join('./node_modules/', filename))) {
+    return {contents: fs.readFileSync(path.join('./node_modules/', filename)).toString()};
+  }
```

Comment:
Since we already do `fs.existsSync` in Config.js, shouldn't we put the path to the file (be it in a dir or in node_modules) and just use that one? So that we don't check twice.

Reply:
Just saw that this is called onMessage. I guess we don't have access to the config object? If we don't disregard my previous comment.

Reply:
ok there are two things:
- this part of the code is run in a separate process altogether, so we don't have access to the same code
- this method is actually a fallback for the solidity compiler and is not related to that anyway, the fallback will be activated when an import (in the solidity code itself, not from embark), is not found